frontend/app/src/main/java/com/chads/vanroomies/ListingsRecyclerData.java

```diff
@@ -2,7 +2,7 @@
 // Reference: https://www.geeksforgeeks.org/recyclerview-using-gridlayoutmanager-in-android-with-example/
 public class ListingsRecyclerData {
     private String title;
-    private int imgId;
+    private String imageString;
```

Comment:
change to imageId?

Reply:
imsageString contains it is an image as a Base 64 encoded String, so I think it should be kept this way (not an id)